pulse_actions/handlers/talos.py

```diff
@@ -0,0 +1,79 @@
+"""
+This module is for the following use case:
+
+ - Talos jobs:
+
+    * Trigger talos jobs twice if they are from PGO build.
+"""
+import logging
+
+from pulse_actions.utils.misc import filter_invalid_builders
+
+from mozci import query_jobs
+from mozci.errors import PushlogError
+from mozci.mozci import (
+    trigger_job
+)
+from mozci.sources import buildjson
+from requests.exceptions import ConnectionError
+
+LOG = logging.getLogger()
+
+def on_event(data, message, dry_run):
+    """Trigger talos jobs twice in PGO builds."""
+    # Cleaning mozci caches
+    buildjson.BUILDS_CACHE = {}
+    query_jobs.JOBS_CACHE = {}
+    payload = data["payload"]
+    status = payload["status"]
+    buildername = payload["buildername"]
+
+    if "pgo" in buildername and ("mozilla-inbound" in buildername or "fx-team" in buildername):
```

Comment:
`get_buildername_metadata` is a slight problem due to https://github.com/mozilla/mozilla_ci_tools/issues/468 . That's why I've used Python's `in` operator to check these three things using the `if` condition @armenzg